src/opening-period/page/CreateNewOpeningPeriodPage.tsx

```diff
@@ -0,0 +1,228 @@
+          type="error">
+          Tarkista toimipiste-id.
+        </Notification>
+      </>
+    );
+  }
+
+  return (
+    <>
+      {submitStatus === 'succeeded' && (
+        <Notification
+          dataTestId="opening-period-added-successfully-notification"
+          position="top-right"
+          autoClose
+          size="small"
+          label="Aukiolojakso tallennettu onnistuneesti"
+          onClose={(): void => setSubmitStatus('init')}
+          type="success">
+          Aukiolojakso tallennettu onnistuneesti
+        </Notification>
+      )}
+      {submitStatus === 'error' && (
+        <Notification
+          position="top-right"
+          autoClose
+          size="small"
+          label="Aukiolojakson lisääminen epäonnistui"
+          onClose={(): void => setSubmitStatus('init')}
+          type="error">
+          Aukiolojakson lisääminen epäonnistui. Yritä myöhemmin uudestaan.
+        </Notification>
+      )}
+      <div className="add-new-opening-period-page-header-row">
+        <ResourceInfo resource={resource} />
+        <h2 className="add-new-opening-period-page-title">
+          Toimipisteen aukiolojakson lisäys
+        </h2>
+      </div>
+      <form
+        id="add-new-opening-period-form"
+        data-cy="add-new-opening-period-form"
+        className="add-new-opening-period-form"
+        onSubmit={handleSubmit(onSubmit)}>
+        <div className="form-content-container">
+          <h3 className="opening-period-section-title">Jakson kuvaus</h3>
+          <label htmlFor="openingPeriodTitle">Aukiolojakson otsikko *</label>
+          <input
+            className="add-new-opening-period-title"
+            type="text"
+            name="openingPeriodTitle"
+            data-cy="openingPeriodTitle"
+            id="openingPeriodTitle"
+            aria-invalid={errors.openingPeriodTitle ? 'true' : 'false'}
+            ref={register({ required: true, maxLength: 100 })}
+          />
+          {errors.openingPeriodTitle &&
+            errors.openingPeriodTitle.type === 'required' && (
+              <span role="alert">Aukiolojakson otsikko on pakollinen</span>
+            )}
+
+          <label htmlFor="openingPeriodOptionalDescription">
+            Jakson valinnainen kuvaus
+          </label>
+          <textarea
+            cols={90}
+            className="opening-period-optional-description"
+            id="openingPeriodOptionalDescription"
+            name="openingPeriodOptionalDescription"
+            ref={register({ maxLength: 255 })}
+          />
+          <h3 className="opening-period-section-title">Ajanjakso</h3>
+          <section className="opening-period-time-period">
+            <Datepicker
+              id="openingPeriodBeginDate"
+              dataCy="openingPeriodBeginDate"
+              labelText="Alkaa"
+              onChange={(value): void => setPeriodBeginDate(value || null)}
+              onBlur={(): void => {
```

Comment:
Are these empty blur-callbacks needed?

Reply:
If I remember correct, they were required by the Datepicker component. I'll check. Though now that we have copied the whole component, we could change them as optional 😁

Reply:
- [ ] Remove unused onBlur methods